vpr/src/route/build_switchblocks.cpp

```diff
@@ -324,10 +327,11 @@ t_sb_connection_map* alloc_and_load_switchblock_permutations(const t_chan_detail
         if (directionality != sb.directionality) {
             VPR_FATAL_ERROR(VPR_ERROR_ARCH, "alloc_and_load_switchblock_connections: Switchblock %s does not match directionality of architecture\n", sb.name.c_str());
         }
+        
```

Comment:
can you please change the for loop that iterates over switchblocks to a range-based for loop that iterate over elements by reference?

Github doesn't allow me to add the comment there (line 323)